experimental/generator-dotnet-yeoman/generators/app/templates/functions/Startup.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Bot.Builder.Runtime.Extensions;
+
+[assembly: FunctionsStartup(typeof(Microsoft.Bot.Builder.Runtime.FunctionHost.Startup))]
+
+namespace Microsoft.Bot.Builder.Runtime.FunctionHost
```

Comment:
Replace namespace with <%= botName %>

Reply:
Or, rather, specifically, any instances of 'Microsoft.Bot.Builder.Runtime.FunctionHost' within this project.